app/src/main/java/src/PiecesMoves/PawnMove.java

```diff
@@ -0,0 +1,92 @@
+package src.PiecesMoves;
+
+import src.Utilities.Chessboard;
+import src.Pieces.Piece;
+import src.Utilities.Position;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PawnMove implements MovePiece {
+
+    /**
+     * Return a List of Positions with all possible moves of Pawn.
+     *
+     * @param piece the Pawn piece
+     * @return movePoints List of Points of moves
+     */
+    public List<Position> getPossibleMoves(final Piece piece) {
+        List<Position> movePositions = new ArrayList<>();
+        int yOneMove = piece.getPosY() + 1;
```

Comment:
Could use final attributes for the pawn moves such as private final int pawnSingleMove = 1
private final int pawnDoubleMove = 1